.github/actions/setup-microceph/action.yml

```diff
@@ -54,6 +54,7 @@ runs:
           sudo snap install microceph --channel "${{ inputs.microceph-channel }}"
           sudo microceph cluster bootstrap
           sudo microceph.ceph config set global osd_pool_default_size 1
+          sudo microceph.ceph config set global mon_allow_pool_size_one true
```

Comment:
what does this do, that `global osd_pool_default_size 1` doesnt do btw?

Reply:
looks like the previous commit https://github.com/canonical/lxd/pull/14044/commits/6a376887db8e93006ac51a88f98fdc4ba8effcf7 needs to go as its been changed here.

Reply:
`osd_pool_default_size` sets the default (unspecified) pool size to 1. `mon_allow_pool_size_one` allows you to explicitly set a pool's size to 1.

Reply:
Commit ordering is fixed too.